### PR TITLE
Contact footer version branch

### PR DESCRIPTION
**Pull request by tsohleDev**
Adding the contact me section and a footer with my social links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -218,13 +219,12 @@ <h3 class="h3">Skills</h3>
...
-            <form action="" class="contact__form">
-                <input type="text" placeholder="your name" name="name" class="name">
-                <input type="email" placeholder="your email" name="email" class="email">
-                <input type="text" placeholder="Enter your message here..." name="message" id="" class="message">
-                <input type="button" value="Start Collaboration" class="buttons contact__button">
+            <form action="https://formspree.io/f/mrgdozvp" method="post" class="contact__form">
+                <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+                <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
+                <textarea type="text" required placeholder="Enter your message here..." name="message" id="" class="message"> </textarea>
+                <input type="submit" maxlength="500" value="Start Collaboration" class="buttons contact__button">
```
Comment (Olamarx) [inline index.html]: - [ ] This maxlength attribute should be used in the textarea, not the submit button as we are not writing anything in the submit.

File: index.html [hunk 2]
```diff
@@ -218,13 +219,12 @@ <h3 class="h3">Skills</h3>
...
             <h2 class="h2">Get Started</h2>
             <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
-            <form action="" class="contact__form">
-                <input type="text" placeholder="your name" name="name" class="name">
-                <input type="email" placeholder="your email" name="email" class="email">
-                <input type="text" placeholder="Enter your message here..." name="message" id="" class="message">
-                <input type="button" value="Start Collaboration" class="buttons contact__button">
+            <form action="https://formspree.io/f/mrgdozvp" method="post" class="contact__form">
+                <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+                <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
```
Comment (Olamarx) [inline index.html]: [OPTIONAL] Nice one adding the pattern attribute but emails should not be case-sensitive. This means it should not check if I input it in lowercase or uppercase. So far I input a valid email then it should work which is the work of the type attribute email. I did use lowercase and it was submitted. I will encourage you to take a second look at it.
### Check the warning message, the user will not even understand the format you are expecting so if you are going to continue with this pattern, you have to add the expected format message, it maybe through the placeholder.
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @tsohleDev, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Form submits as expected ✔️ 
- Linter passed ✅ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @tsohleDev 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

Form submits as expected ✔️
Linter passed ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**